lib/active_model_serializers/adapter/json_api.rb

```diff
@@ -234,6 +235,12 @@ def resource_objects_for(serializers)
         [@primary, @included]
       end
 
+      def paginate?
```

Comment:
I actually wondered if this shouldn't be the concern of `PaginationLinks` e.g.

```ruby
pagination_links = PaginationLinks.new(serializer, serialization_context)
hash[:links] = pagination_links.add_to(hash) # check if paginated and add the links in such case
```

the same way as the `Jsonapi` class does